app/config.py

```diff
@@ -155,6 +155,7 @@ class Config(object):
     MOU_SIGNED_ON_BEHALF_SIGNER_RECEIPT_TEMPLATE_ID = 'c20206d5-bf03-4002-9a90-37d5032d9e84'
     MOU_SIGNED_ON_BEHALF_ON_BEHALF_RECEIPT_TEMPLATE_ID = '522b6657-5ca5-4368-a294-6b527703bd0b'
     MOU_NOTIFY_TEAM_ALERT_TEMPLATE_ID = 'd0e66c4c-0c50-43f0-94f5-f85b613202d4'
+    CONTACT_US_TEMPLATE_ID = '8ea9b7a0-a824-4dd3-a4c3-1f508ed20a69'
```

Comment:
Rest of PR looks good, the only open issue is that the CONTACT US template needs to be generated using a database migration so that if you have to stand the product up from scratch you can. Take a look here how GDS does it:

https://github.com/cds-snc/notification-api/blob/master/migrations/versions/0298_add_mou_signed_receipt.py

Note that they hard code the ID so it remains consistent.